documentation/modules/ROOT/pages/integration-with-aap.adoc

```diff
@@ -0,0 +1,176 @@
+
+Next, we create our PolicyAutomation in the form of an AnsibleJob, binding it to our defined Policy. Upon our Policy binding to a spoke cluster, RHACM will trigger our PolicyAutomation defined AnsibleJob, which starts an automation job on the Automation Controller, passing a list of targeted clusters (the clusters PlacementRule has bound to) as an extra variable. 
+
+Now, the same Policy that triggered the Ansible job also checks for the necessary cluster state seeking to be achieved by the completion of the job. Only when the job completes successfully and the cluster reaches a compliant state does the automation policy reach a compliant status.
+
+[#workflow-overview]
+== Workflow Overview
+
+Now that we are familiar with AAP in OpenShift and the necessary components for integration, let's walk through an Ansible integration workflow.
+
+[#general-workflow]
+=== General AAP Integration Workflow
+
+At a high level, the steps to AAP integration with 5G RAN ZTP deployments are as follows:
+
+* Mirror AAP operator to disconnected registry (outside the scope of this lab, see the _Ansible Automation Platform mirroring_ section of this link:https://www.ansible.com/blog/openshift-and-ansible-bridging-the-automation-gap-for-5g-and-beyond-networks-part-1[blog])
+* Install AAP operator, deploy AutomationHub and AutomationController (already running on OpenShift cluster, see the _Ansible Automation Platform operator installation_ section of this link:https://www.ansible.com/blog/openshift-and-ansible-bridging-the-automation-gap-for-5g-and-beyond-networks-part-1[blog] and the CRD examples for link:integration-with-aap.html#automation-hub[AutomationHub] and link:integration-with-aap.html#automation-controller[AutomationController])
+* Upload necessary Ansible collections to Automation Hub (already uploaded, see uploaded collections at automation-hub-aap.apps.hub.5g-deployment.lab and instructions for this process in the link:lab-environment.html#upload-collections-to-aap-hub[Lab Environment] section)
+* Configure Automation Controller with Ansible automation resources
+* Install MultiClusterHub add-ons
+* Deploy spoke clusters
+* Deploy ManagedServiceAccount, Policy, PlacementBinding, PlacementRule, and PolicyAutomation for a governance-based hook into AAP
+* Spoke cluster(s) reach state where Policy applies, Ansible job is triggered by PolicyAutomation on each cluster, Ansible job completes successfully, Policy moves to compliant for that cluster
+
+This lab will cover the process starting with configuring the Automation Controller and ending with a successful Ansible job and a compliant automation Policy for a deployed cluster.
+
+[#lab-workflow]
```

Comment:
As I put in the other docs, I think this is outside the scope of the lab and should be done in the automation. No need to explain the specific steps to the user since they will not be doing them.